capstone/src/main/webapp/youtubeScript.js

```diff
@@ -347,3 +357,47 @@ async function getKeywordSearchResults() {
   });
   document.getElementById('search-type').innerHTML = "Keyword Search";
 }
+
+/** Prepares CSV download*/
+function prepareDownload(sheetHeader, sheetData, sheetName) {
+  const header = sheetHeader.join(',') + '\n';
+  let csv = header;
```

Comment:
Is it possible to combine these two lines?
`let csv = sheetHeader.join(',') + '\n';`

Reply:
addressed in "address Daniel's and Chris's nits"